wouso/resources/templates/top/sidebar.html

```diff
@@ -32,6 +32,8 @@
     {% endif %}
     <a href="{% url quest_history %}" class="goto">{% trans 'Quest History' %}</a>
     <br/>
+    <a href="{% url qotd_history_view %}" class="goto">{% trans 'Qotd History' %}</a>
```

Comment:
In 2 lines above, there is the same `if` content, but a different test variable. Please modify the variable above, and then request a review again.